### PR TITLE
docs: revise help screens

### PR DESCRIPTION
**Pull request by inventarSarah**
- revised help screens for `globalping --help` and `globalping [command] --help`
- did not update help text for the `completion` command as it seems to be generated by the used library

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -68,23 +68,23 @@ func NewRoot(
...
-	For example, the partial or full name of a continent, region (e.g eastern europe), country, US state, city or network
-	Or use [@1 | first, @2 ... @-2, @-1 | last | previous] to run with the probes from previous measurements.`)
-	flags.IntVarP(&ctx.Limit, "limit", "L", ctx.Limit, "Limit the number of probes to use")
-	flags.BoolVarP(&ctx.ToJSON, "json", "J", ctx.ToJSON, "Output results in JSON format (default false)")
-	flags.BoolVarP(&ctx.CIMode, "ci", "C", ctx.CIMode, "Disable realtime terminal updates and color suitable for CI and scripting (default false)")
-	flags.BoolVar(&ctx.ToLatency, "latency", ctx.ToLatency, "Output only the stats of a measurement (default false). Only applies to the dns, http and ping commands")
-	flags.BoolVar(&ctx.Share, "share", ctx.Share, "Prints a link at the end the results, allowing to vizualize the results online (default false)")
+	flags.StringVarP(&ctx.From, "from", "F", ctx.From, `Specify the probe location by providing a comma-separated list of location values, such as continents, regions, countries, US states, cities, or networks.
```
Comment (MartinKolarik) [inline cmd/root.go:81]: Please follow these conventions of the underlying library for consistency:
 - no periods at the end for commands/flags
 - flag descriptions not capitalized (command descriptions stay capitalized as they are)

File: cmd/root.go [hunk 2]
```diff
@@ -68,23 +68,23 @@ func NewRoot(
...
+	Alternatively, use [@1 | first, @2 ... @-2, @-1 | last | previous] to run with the probes from your previous measurements. (default "world")`)
+	flags.IntVarP(&ctx.Limit, "limit", "L", ctx.Limit, "Define the number of probes to use. (default 1)")
```
[the remark this answers is not in the document]
Comment (inventarSarah) [reply cmd/root.go]: Updated the two flag descriptions and removed the defaults; I also reviewed the other flags but didn't find another case where it added the default text automatically

File: cmd/root.go (hunk 1 above)
Comment (inventarSarah) [reply cmd/root.go]: Thank you -- I hope I covered everything you found:

- updated the flags
  - start lowercase
  - removed periods (I needed to rewrite some descriptions for this) 
- updated command `Short` descriptions
  - removed period

**Comment by MartinKolarik** [comment]
Looks good, thanks!

**Pull request merged**